queries/goctl/folds.scm

```diff
@@ -0,0 +1,21 @@
+; Fold types
+(typeGroupSpec) @fold
+
+(typeSingleSpec) @fold
+
+(typeStruct) @fold
+
+; Fold services
+(serviceSpec) @fold
+
+(serviceBody) @fold
+
+; Fold import groups
+(importStatement) @fold
```

Comment:
Please group all of the folds in this file; the comments are mostly unnecessary because the node names are self-descriptive.

> "The proper use of comments is to compensate for our failure to express ourself in code." - Uncle bob

Reply:
done.

Reply:
Also sorry to nitpick but would you mind putting all of these in one grouping? E.g.
```scheme
[
  (typeGroupSpec)
  (typeSingleSpec)
  ; ...
  (structType)
] @fold 
```
If you look at our other fold queries they are mostly all like that

Reply:
> Also sorry to nitpick but would you mind putting all of these in one grouping? E.g.
> 
> ```scheme
> [
>   (typeGroupSpec)
>   (typeSingleSpec)
>   ; ...
>   (structType)
> ] @fold 
> ```
> 
> If you look at our other fold queries they are mostly all like that

off course, done.